tools/ports/harfbuzz.py

```diff
@@ -18,11 +18,17 @@ def create():
       freetype_include = os.path.join(freetype_dir, 'include')
       freetype_include_dirs = freetype_include + ';' + os.path.join(freetype_include, 'config')
 
-      shared.Building.configure(['cmake', '-H' + source_path, '-B' + dest_path,
-        '-DHB_HAVE_FREETYPE=ON', '-DFREETYPE_LIBRARY=' + freetype_lib,
+      shared.Building.configure([
+        'cmake',
+        '-B' + dest_path,
+        '-H' + source_path,
+        '-DCMAKE_BUILD_TYPE=Release',
+        '-DCMAKE_INSTALL_PREFIX=' + dest_path,
         '-DFREETYPE_INCLUDE_DIRS=' + freetype_include_dirs,
-        '-DCMAKE_BUILD_TYPE=Release'])
-      shared.Building.make(['make', '-C' + dest_path])
+        '-DFREETYPE_LIBRARY=' + freetype_lib,
+        '-DHB_HAVE_FREETYPE=ON'
+      ])
+      shared.Building.make(['make', '-C' + dest_path, 'install'])
```

Comment:
does `make install` also do the normal `make` step of building the source files? lgtm if so.

Reply:
Yes, it does. I originally had both `make` and `make install` here. Tested that it works after I deleted Harfbuzz from `.emscripten_cache` and `.emscripten_ports`.

Reply:
Great, thanks for verifying.